File: tests/fixtures/graph/go/sample.go
package sample

import (
	"fmt"
	alias "strings"
)

type Widget struct{}

func Render(w Widget) string {
	helper()
	w.Draw()
	fmt.Println(alias.TrimSpace(" rendering "))
	return "ok"
}

func (w Widget) Draw() {}

func helper() {}

File: tests/fixtures/graph/rust/sample.rs
use std::collections::HashMap;

pub struct Widget {
    values: HashMap<String, String>,
}

pub enum Mode {
    Compact,
}

pub trait Renderable {
    fn render(&self) -> String;
}

impl Widget {
    pub fn render(&self) -> String {
        helper();
        self.draw();
        println!("rendering");
        String::from("ok")
    }

    fn draw(&self) {}
}

fn helper() {}
